_pages/en_US/include/addproxy.txt

```diff
@@ -12,4 +12,4 @@ A visual guide is available for this section [here](/images/screenshots/set-prox
 1. Click OK, then click Save
 1. When prompted, click "Test" to perform the connection test
     + The test should succeed
-1. Click "OK" to continue
+1. Click "OK" to continue
```

Comment:
this is the coolest change of all time

Reply:
Pretty sure this adds a newline to EOF, I don't particularly mind that because realistically it should be there